### PR TITLE
fix: getStatuses does not reject when data is not returned

### PR DESCRIPTION
**Pull request by felipeguimaraes2**
getStatuses does not reject when data is not returned

### CODE REVIEW

File: packages/adapter/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# @node-escpos/core
+# @node-escpos/adapter
```
Comment (dohooo) [inline packages/adapter/README.md:1]: What's this?

Comment (dohooo) [reply packages/adapter/README.md:1]: And others.

**Comment by dohooo** [comment]
Thanks for your contribution, and you can add a changeset for these changes.

File: packages/adapter/README.md (hunk 1 above)
Comment (felipeguimaraes2) [reply packages/adapter/README.md:1]: I forked from your main branch (it's behind your changeset branch), so they are your commits in the main branch.

Comment (dohooo) [reply packages/adapter/README.md]: Oh, right.

**Comment by dohooo** [comment]
I'll merge it when you add the changeset.

**Review by dohooo — APPROVED** [review]
LGTM

**Pull request merged**